core/lib/Thelia/TaxEngine/TaxType/FeatureFixAmountTaxType.php

```diff
@@ -46,7 +47,9 @@ public function fixAmountRetriever(Product $product)
             ->findOne();
 
         if (null !== $query) {
-            $taxAmount = $query->getFreeTextValue();
+            if ( (1 == $taxAmount = $query->getFreeTextValue()) && !is_null($query->getFeatureAvId()) ) {
```

Comment:
Its seems that already was in 2.2.0-beta1.php
Let's count that no one have `NULL` in `feature_av_id` then ?